workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/EntryPoint.cs

```diff
@@ -106,6 +176,7 @@ private string GenerateBundle()
                 $"--descriptor_set_out=\"{descriptorPath}\""
             }.Union(inputPaths).ToList();
 
+            logger.Info("Generating schema bundle and descriptor");
```

Comment:
Would be nice to `logger.Trace` the command + arguments that we are about to execute